src/uct/tcp/tcp.h

```diff
@@ -56,6 +56,8 @@
 #define UCT_TCP_EP_CTX_CAPS                  (UCS_BIT(UCT_TCP_EP_CTX_TYPE_TX) | \
                                               UCS_BIT(UCT_TCP_EP_CTX_TYPE_RX))
 
+#define UCT_TCP_EP_ID_MAX                    UINT16_MAX
```

Comment:
does it mean we don't support more that 64k connections during the lifetime of the iface?

Reply:
yes, the intent was to support 64k simultaneously
added the comment and fixed

Reply:
what if there is a loop of 64k ep_create/destroy?